week4/src/pages/components/search.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ChangeEvent, FormEvent } from 'react'
+
+interface searchProps {
+  search: string
+  type: string
+  change: (event: ChangeEvent<HTMLInputElement>) => void
+  submit: (event: FormEvent<HTMLFormElement>) => void
+  selectChange: (event: ChangeEvent<HTMLSelectElement>) => void
+}
+
+const Search: React.FC<searchProps> = ({
+  search,
+  type,
+  change,
+  submit,
+  selectChange,
+}) => {
+  return (
+    <div className="searchBar">
+      <form onSubmit={submit} className="form">
+        <input
+          className="search"
+          type="text"
+          placeholder="Search Pokemon"
+          value={search}
+          onChange={change}
+        />
+        <button className="button" type="submit">
+          Search
+        </button>
+      </form>
+      <div>
+      <select id="type" className="select" value={type} onChange={selectChange} >
+          <option value="All">All</option>
```

Comment:
Instead of adding all the option you have the options in an array and loop them over to create the option